src/main/resources/models/security.hjson

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Lets remove this file for now based on the comment in elide-standalone. We dont support rules completely yet and the roles we are creating here are not being used in the Tables.hjson.